docs/source/examples/CZML Tutorial.mystnb

```diff
@@ -104,21 +104,33 @@ extractor.add_orbit(molniya,
              )
 ```
 
-You can now export the extractor packets by simply calling ``extractor.packets`` and load it to the Cesium app as described [here](https://github.com/poliastro/cesium-app)
-
-```{code-cell}
-extractor.packets
-```
+You can now export the extractor packets by simply calling ``extractor.packets`` and load it to the Cesium app as described [here](https://github.com/poliastro/cesium-app):
+
+```python    
+  >>> extractor.packets
+  [{
+     "id": "document",
+     "version": "1.0",
+     "name": "document_packet",
+     "clock": {
+         "interval": "2000-01-01T12:00:00Z/2000-01-01T23:59:35Z",
+         "currentTime": "2000-01-01T12:00:00Z",
+         "multiplier": 60,
+         "range": "LOOP_STOP",
+         "step": "SYSTEM_CLOCK_MULTIPLIER"
+     }
+ }, [...]
+  ```
```

Comment:
Hey @xlxs4 ... Have you tested this? I don't think this may work as the type of documentation we are working with is not completely in Markdown format ... It may not natively allow for normal markdown syntax